crates/cargo-winrt/src/main.rs

```diff
@@ -215,10 +237,10 @@ impl Install {
                 })
                 .collect::<anyhow::Result<Vec<DependencyDescriptor>>>()?
         };
-        debug!(
-            "{} {} nuget dependencies",
-            console::style("Installing").green().bold(),
-            dependency_descriptors.len()
+
+        print_verbose_status(
+            "Installing",
+            format_args!("{} nuget dependencies", dependency_descriptors.len()),
```

Comment:
The `format_args!` are *probably* overkill, but the previous `debug!` macro for verbose output wouldn't evaluate the formatted string unless `-v` was actually specified, so I wanted to at least prove that this change could maintain that behavior if desired.